content/html-css/sprints/1/prep/index.md

```diff
@@ -7,11 +7,40 @@ weight = 1
 backlog= 'Module-HTML-CSS'
 backlog_filter= 'Week 1'
 [[blocks]]
+name="spaced-repetition"
+[[blocks]]
+name="what-is-html"
+src="module"
+[[blocks]]
 name="Transferable skills"
 src="https://cyf-pd.netlify.app/blocks/prep-transferable-skills/readme/"
 [[blocks]]
 name="Feedback"
 src="https://cyf-pd.netlify.app/blocks/prep-for-feedback/readme/"
+[[blocks]]
+name="Playlist"
+src="https://www.youtube.com/watch?v=mBmX7944vd0&list=PLozA7cloMbPgk5gK8His2CeuJTKofypbV&index=7"
+[[blocks]]
+name="Bikes for Refugees"
+src="https://github.com/CodeYourFuture/bikes-for-refugees/readme"
```

Comment:
In the rendered text for this block, we have the text "Clone this repository onto your machine", but there's no context as to what repository it is. In the README itself, it can be assumed you have that context, but if we're just in-lining the content into another page, that context is lost, and we'll need to add it somehow...

Also, I just filed https://github.com/CodeYourFuture/curriculum/issues/202 about migrating the git cheat-sheet and https://github.com/CodeYourFuture/curriculum/issues/203 about updating the branch creation instructions